docs/openapi/resources/credential.yml

```diff
@@ -6,6 +6,7 @@ get:
   tags:
     - Credentials
   security:
+    - {}
```

Comment:
```suggestion
```

Reply:
ahh, this is wrong... revocation lists require this to be open to any verifier...

Reply:
I need to read more carefully.